bundles/org.openhab.binding.deconz/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -36,8 +36,20 @@
 
 	<config-description uri="thing-type:deconz:sensor">
 		<parameter name="id" type="text" required="true">
-			<label>Sensor ID</label>
-			<description>The deCONZ bridge assigns an integer number ID to each sensor.</description>
+			<label>Device ID</label>
+			<description>The deCONZ bridge assigns an integer number ID to each device.</description>
+		</parameter>
+	</config-description>
+
+
+	<config-description uri="thing-type:deconz:light">
+		<parameter name="id" type="text" required="true">
+			<label>Device ID</label>
+			<description>The deCONZ bridge assigns an integer number ID to each device.</description>
+		</parameter>
+		<parameter name="transitiontime" type="decimal" required="false" min="0" unit="s">
```

Comment:
As far as I understood the REST API documentation the transition time is only used if "effect" is activated. So it is only needed if also a channel for "effect" exists.

Reply:
I tried it with a dimmable IKEA bulb and it takes 20s to turn it on if I send 200 together with the brightness, so no matter what the documentation says, it works